File: golang/control_flows_and_loops/control_flows/control_flows_01.go
package main

import (
	"fmt"
	"math/rand"
	"time"
)

func main() {

	//In order to avoid : Go rand.Intn return the same value on each run : https://stackoverflow.com/questions/39529364/go-rand-intn-same-number-value
	rand.Seed(time.Now().UnixNano())

	//Give me a random number between 0 and 100
	number := rand.Intn(100)
	fmt.Println(number)

	//If example
	if number < 50 {
		fmt.Println("Small")
	} else {
		fmt.Println("Big")
	}

}

File: python/control_flows_and_loops/control_flows/control_flows_01.py
import random

# Give me a random number between 0 and 100
some_random_number = random.randint(0,100)
print(some_random_number)

# If example
if some_random_number < 50:
	print("Small")
else:
	print("Big")


# elif - else-if
if some_random_number < 50:
	print("Small")
elif some_random_number < 80:
	print("Medium")
else:
	print("Big")




